lib/src/style/QlementineStyle.cpp

```diff
@@ -861,8 +861,31 @@ void QlementineStyle::drawPrimitive(PrimitiveElement pe, const QStyleOption* opt
         const auto focus =
           widgetHasFocus && selection == SelectionState::Selected ? FocusState::Focused : FocusState::NotFocused;
         const auto active = getActiveState(itemState);
-        const auto& color = listItemBackgroundColor(mouse, selection, focus, active, optItem->index, w);
-        p->fillRect(rect, color);
+        if (const QStyleOptionViewItem *vopt = qstyleoption_cast<const QStyleOptionViewItem*>(opt)) {
```

Comment:
Please use `auto` as much as possible 😊
```suggestion
        if (const auto *vopt = qstyleoption_cast<const QStyleOptionViewItem*>(opt)) {
```

Reply:
Btw, why do you cast again `opt`? It's already made a few lines above?
So the code in the `else` statement will never be called... So you broke the styling made by Qlementine.